zebra-chain/src/block/tests.rs

```diff
@@ -248,24 +264,18 @@ fn block_limits_single_tx() {
         .zcash_serialize(&mut data)
         .expect("block should serialize as we are not limiting generation yet");
 
-    // Check size
-    assert_eq!(data.len(), 1999972);
-
```

Comment:
Issue: The block size is the purpose of the test, so let's make sure `data.len() <= MAX_BLOCK_BYTES`.